bob/main.py

```diff
@@ -274,19 +288,20 @@ def replace_weather_description_with_emojis(description):
             description = j[1] + " " + j[0]
     return description
 
+
 def wind_direction(degrees):
     directions = ['pohjoisesta','koillisesta','idästä','kaakosta','etelästä','lounaasta','lännestä','luoteesta']
     cardinal = round(degrees / (360/len(directions)))
     return directions[cardinal % len(directions)]
 
 
-def low_probability_reply(update, context, int=0): # added int argument for unit testing
-    if int == 0:
-        random_int = random.randint(1,10000) # 0,01% probability
+def low_probability_reply(update, context, integer=0):  # added int argument for unit testing
+    if integer == 0:
+        random_int = random.randint(1, 10000)  # 0,01% probability
     else:
-        random_int = int
+        random_int = integer
     if random_int == 1:
-        reply_text = "Vaikuttaa siltä että olette todella onnekas " + "\U0001F340" # clover emoji
+        reply_text = "Vaikuttaa siltä että olette todella onnekas " + "\U0001F340"  # clover emoji
```

Comment:
Could be actual emoji here for better readability, like we have in weather functionality.